data/flags/man_made/flagpole.json

```diff
@@ -3432,6 +3432,30 @@
         "subject:wikidata": "Q804"
       }
     },
+    {
+      "displayName": "Pan-African",
+      "id": "panafrican-e5dc93",
+      "locationSet": {"include": ["001"]},
+      "matchNames": [
+        "afro-american",
+        "bendera ya taifa",
+        "black liberation",
+        "international african",
+        "marcus garvey",
+        "rbg",
+        "red black green",
+        "unia",
+        "universal african"
+      ],
+      "tags": {
+        "flag:name": "Pan-African",
+        "flag:type": "cultural",
+        "flag:wikidata": "Q2029729",
+        "man_made": "flagpole",
+        "subject": "Pan-Africanism",
+        "subject:wikidata": "Q282739"
```

Comment:
Based on the `flag:type=cultural`, Pan-Africanism seemed like an appropriate `subject`, whereas [Garveyism](https://en.wikipedia.org/wiki/Garveyism) would seem like a more appropriate `subject` for a `political` or `organisation` flag. That said, I’m open to changing it or removing it if there’s variation in what the flag represents when flown.